LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-and-navigate-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-FAIL Using intercept() then navigate() in the ensuing currententrychange should abort the finished promise (but not the committed promise) assert_equals: expected "#2" but got ""
+FAIL Using intercept() then navigate() in the ensuing currententrychange should abort the finished promise (but not the committed promise) assert_unreached: Should have rejected: undefined Reached unreachable code
```

Comment:
Do you understand this failure?

Reply:
I think I do, the test now gets past the location.hash assert line, but either we lack the logic to abort the back navigation because of the new navigation to "#2", or we do have the logic but it is not triggered (at the right time).  We do know some of our timing is off since we did not implement the "wait for all promises" logic yet, that could be related. Anyway, we'll revisit this in follow up PRs.